### DISCUSSION TITLE
OSCR shows no signs on life

### DISCUSSION THREAD

**Discussion by tsears9109** [sanni/cartreader#787] [Need help]
I finished soldering everything about two days ago and have been trying to troubleshoot ever since. I'm using the HW4 Rev3 Board, Everything is installed correctly according to the wiki. 1k resistor for the LED, LED is installed correctly, Arduino isn't bad (I was able to flash it before modifying it with no issues), screen pads have been bridged, etc. etc. I also don't have any bad solder joints. The screen doesn't turn on at all, nor do the lights on the arduino. The status LED does come on when I power on the OSCR, though.

**Comment by sanni** [comment]
Can you still flash it now while it's assembled?
Does it create a OSCR_LOG.txt on  the SD card?

**Reply by tsears9109** [reply]
Unfortunately not, the Mega 2560 does not show up as a device on the PC now that it requires power from the mainboard. No new files are on the SD card either, it's just the database files.

**Reply by sanni** [reply]
Sounds like a short-circuit between GND and VCC to me. 
You need to use a multimeter to figure out on which PCB the short is happening.

One possible reason is the NES pins touching: https://github.com/sanni/cartreader/discussions/637#discussioncomment-4232864

**Reply by tsears9109** [reply]
Figured out the issue. I didn't have J1 soldered on HW5 Rev4

**Reply by sanni** [reply]
Thanks for reporting back. I will add that problem and the solution to the wiki. 👍

**Reply by Ancyker** [reply]
> Thanks for reporting back. I will add that problem and the solution to the wiki. 👍

Should we maybe change that to a default closed jumper you need to cut to open if you want to do serial or something?

**Reply by tsears9109** [reply]
I don't personally think that would be necessary, in my case I just didn't solder the jumper because the guide specifically says it's only necessary so do so for the Rev3, so changing that in the guide should be good enough. I don't think many people would be too keen about irreversibly damaging their boards.

**Reply by Ancyker** [reply]
> I don't personally think that would be necessary, in my case I just didn't solder the jumper because the guide specifically says it's only necessary so do so for the Rev3, so changing that in the guide should be good enough. I don't think many people would be too keen about irreversibly damaging their boards.

Ah, I assume you are the one that asked in Discord then, lol. But no, it's a common issue. I forgot the damn jumper a few times, and others have forgotten to do it too. And it's generally going to be closed more than it's open, so it'd make sense to close it on the PCB itself and have someone who didn't want it closed just cut the trace. It's one less thing to go wrong. 

I don't think there's even a stated way to do HW5 as a serial-only reader, is there? Without the LCD module, there's no 3.3V rail available. The cost of a PCB to add a 3.3V rail and to components would approach the cost of the LCD module, so I'm not sure it'd even be worth it.

**Reply by sanni** [reply]
Good catch, I removed the "only for Rev3" from the wiki.

The purpose of the jumper is to isolate the VCC of the Mega Pro and the VCC of the rest of the Cart Reader for troubleshooting when you have a short between GND and VCC. If you open the jumper you can then probe the Mega Pro and the rest separately. 
It has nothing to do with Serial.

It can be removed as the same can be achieved by simply unplugging the Mega Pro. The two other jumpers in close vicinity can also be removed, they don't serve a purpose anymore.

**Reply by Ancyker** [reply]
To be clear, this is what I mean by closing that jumper:

![image](https://github.com/sanni/cartreader/assets/7105333/05ad14f1-c63a-4a7b-bf00-e8d5acf9e2b6)
![image](https://github.com/sanni/cartreader/assets/7105333/c66ea737-e811-4a54-82a7-4cf93c1ea00d)
![image](https://github.com/sanni/cartreader/assets/7105333/8ae83308-f3c1-498f-87d3-1323c583d149)

**Reply by Ancyker** [reply]
> Good catch, I removed the "only for Rev3" from the wiki.
> 
> The purpose of the jumper is to isolate the VCC of the Mega Pro and the VCC of the rest of the Cart Reader for troubleshooting when you have a short between GND and VCC. If you open the jumper you can then probe the Mega Pro and the rest separately. It has nothing to do with Serial.
> 
> It can be removed as the same can be achieved by simply unplugging the Mega Pro. The two other jumpers in close vicinity can also be removed, they don't serve a purpose anymore.

I can remove them if you want, I was about to order a board for testing today for Rev5 anyway.

**Reply by sanni** [reply]
Yes remove all three jumpers. And replace JP1 with just a normal trace without any jumper.

**Reply by Ancyker** [reply]
> (...) replace J1 with just a normal trace without any jumper.

So just VCC, no more VDDA, right?

Edit:
![image](https://github.com/sanni/cartreader/assets/7105333/db41ef3a-b8e6-4d91-b7ff-ecb9a426f887)

**Reply by sanni** [reply]
Yes only VCC.

The initial purpose of the 3 jumpers was to either run the Mega Pro at VCC, which switches between 3V and 5V(jumper J1 and jumper on LCD closed) or on fixed 5V(j1 and lcd jumper open, the other two jumpers closed) if the Arduino can't do 3V @16MHz.

This was for testing purpose only. Not needed anymore.

![IMG_2352](https://github.com/sanni/cartreader/assets/19169494/dec77f2d-0987-41ab-80a8-40d632bcfcea)

And one more thing. On the back side of the On/Off Switch it says "5V USB · 5V". It would be better if it just says "ON · OFF"

**Reply by Ancyker** [reply]
How's this?

Back:
![image](https://github.com/sanni/cartreader/assets/7105333/4b0a9e1d-dc39-482d-a77a-c82444f8ea7d)

Front:
![image](https://github.com/sanni/cartreader/assets/7105333/28729d89-cf2d-4911-b993-1d1b186ebb0d)

**Reply by sanni** [reply]
First picture, top left, the Mega Pro VCC pins are still named VDDA instead of VCC.
Also first picture the pin names on the clock gen are turned 180° compared to the rest of the pin names on the board.
Otherwise it looks great. 👍

**Reply by Ancyker** [reply]
> First picture, top left, the Mega Pro VCC pins are still named VDDA instead of VCC. Also first picture the pin names on the clock gen are turned 180° compared to the rest of the pin names on the board. Otherwise it looks great. 👍

How is it now?
![image](https://github.com/sanni/cartreader/assets/7105333/666ca0d5-a33e-42a6-99eb-7da633a4f0b8)

**Reply by sanni** [reply]
Perfect 👍
